build/Clean.targets

```diff
@@ -0,0 +1,29 @@
+<Project>
+  <Target Name="Clean">
+    <ItemGroup>
+      <FindDirectoryNamesToRemove Include="bin;obj"/>
+    </ItemGroup>
+    <ItemGroup>
+      <DirectoriesToRemove Include="$(TopDirectory)_build"/>
+      <DirectoriesToRemove Include="$(TopDirectory)_install"/>
+      <DirectoriesToRemove Include="$(TopDirectory)_artifacts"/>
+    </ItemGroup>
+
+    <PropertyGroup>
+      <FindRoot>$(TopDirectory.TrimEnd('/').TrimEnd('\\'))</FindRoot>
+      <FindArguments>@(FindDirectoryNamesToRemove -> '-name %(Identity)', ' -or ')</FindArguments>
+    </PropertyGroup>
+    <Exec
+      ConsoleToMSBuild="true"
+      StandardOutputImportance="low"
+      Command="find '$(FindRoot)' -type d -and '(' $(FindArguments) ')'">
```

Comment:
Don't we need a Windows equivalent of this? Leaving note because we don't test `Clean` in the build and don't want it to be forgotten.

Reply:
Yes